test/benchmark/BenchmarkIngestWithNuclio_test.go

```diff
@@ -0,0 +1,148 @@
+package benchmark
```

Comment:
I don't think it's proper for benchmark to be used both as a source root and a package.